Django_GAE/static/js/guestbook/views/GuestbookWidget.js

```diff
@@ -0,0 +1,99 @@
+__author__ = 'buitheanh'
+
+define([
+		   "dojo/_base/declare",
+		   "dojo/cookie",
+		   "dojo/_base/lang",
+		   "dojo/dom-style",
+		   "dojo/on",
+		   "dojo/_base/array",
+		   "dojo/dom-construct",
+		   "dijit/form/Textarea",
+		   "dijit/form/ValidationTextBox",
+		   "dijit/form/Button",
+		   "dijit/form/Form",
+		   "./_ViewBaseMixin",
+		   "./GreetingWidget",
+		   "../store/GreetingStore",
+		   "dojo/text!./templates/GuestbookTemplate.html",
+		   "dojo/domReady!"
+	   ], function(declare, _cookie, lang, domStyle, on, array, contruct, textarea, textbox,
+				   button, from, _ViewBaseMixin, GreetingWidget, storeApi, template){
+	return declare("GuestbookWidget", [_ViewBaseMixin], {
+		// Some default values for our author
+		// These typically map to whatever you're passing to the constructor
+		templateString: template,
+		bookName: "de_name",
+
+		postCreate: function(){
+			this.inherited(arguments);
+			this.loadGreeting();
+			this.own(
+				on(this.submitButton,"click", lang.hitch(this, "_submit")),
+				on(this.changeButton,"click", lang.hitch(this, "_changeGuestbook"))
+			);
+		},
+
+		_changeGuestbook:function(){
+			var value = dijit.byId('addForm').get('value');
```

Comment:
lấy value guestbookName mà lại lấy từ `addForm ` ? 
thử dùng `this.guestbookName.get('value');`